get-poetry.py

```diff
@@ -903,7 +903,12 @@ def get_unix_profiles(self):
 
         if "zsh" in SHELL:
             zdotdir = os.getenv("ZDOTDIR", HOME)
-            profiles.append(os.path.join(zdotdir, ".zprofile"))
+            zprofile = os.path.join(zdotdir, ".zprofile")
+            zshrc = os.path.join(zdotdir, ".zshrc")
+            if os.path.isfile(zprofile):
+                profiles.append(zprofile)
+            else:
+                profiles.append(zshrc)
```

Comment:
Is `.zshrc` more common than `.zprofile`?

Reply:
For me the zsh installation only created a `.zshrc` others have ran into this issue as well (#507)

Reply:
Just another voice: I've only ever had/used a .zshrc file, so I just had this issue trying to install.

Reply:
Also just want to mention that I have only ever had a `.zshrc` and I think it is worth having the measure in place to use `.zshrc` if `.zprofile` doesn't exist

Reply:
@abn Any update?